modopt/tests/test_opt.py

```diff
@@ -119,6 +131,18 @@ def test_forward_backward(self):
         npt.assert_array_equal(self.fb2.x_final, self.data1,
                                err_msg='Incorrect ForwardBackward result.')
 
+        npt.assert_array_equal(self.fb3.x_final, self.data1,
```

Comment:
I realized I also needed to check the result of the algorithm for the lazy start strategies, I don't think it's too much of a mess to have it in this PR as well but lmk.

Reply:
No problem